hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/initialization/BlocklistAccountCreator.java

```diff
@@ -0,0 +1,169 @@
+            final var genesisAccountId = AccountID.newBuilder()
+                    .setRealmNum(0)
+                    .setShardNum(0)
+                    .setAccountNum(2)
+                    .build();
+
+            final var newId = ids.newAccountId(genesisAccountId);
+            if (accounts.contains(newId)) {
+                continue;
+            }
+
+            final var account = blockedAccountWith(evmAddress);
+            accounts.put(newId, account);
+            blockedAccountsCreated.add(account);
+            aliasManager.link(evmAddress, EntityNum.fromAccountId(newId));
+        }
+    }
+
+    private HederaAccount blockedAccountWith(ByteString evmAddress) {
+        final var expiry = properties.getLongProperty(BOOTSTRAP_SYSTEM_ENTITY_EXPIRY);
+        final var account = new HederaAccountCustomizer()
+                .isReceiverSigRequired(true)
+                .isDeclinedReward(true)
+                .isDeleted(false)
+                .expiry(expiry)
+                .memo(BLOCKLIST_ACCOUNT_MEMO)
+                .isSmartContract(false)
+                .key(getGenesisKey())
+                .autoRenewPeriod(expiry)
+                .alias(evmAddress)
+                .customizing(accountSupplier.get());
+        try {
+            account.setBalance(ZERO_BALANCE);
+        } catch (final NegativeAccountBalanceException e) {
+            throw new IllegalStateException(e);
+        }
+        return account;
+    }
+
+    private JKey getGenesisKey() {
+        if (genesisKey == null) {
+            // Traditionally the genesis key has been a key list, keep that way to avoid breaking
+            // any clients
+            genesisKey = asFcKeyUnchecked(Key.newBuilder()
+                    .setKeyList(KeyList.newBuilder().addKeys(asKeyUnchecked(genesisKeySource.get())))
+                    .build());
+        }
+        return genesisKey;
+    }
+
+    private List<String> getPrivateKeyBlocklist() {
+        final var inputStream = getClass().getClassLoader().getResourceAsStream("evm-addresses-blocklist.txt");
+        final var reader = new BufferedReader(new InputStreamReader(inputStream));
+        return reader.lines().toList();
+    }
+
+    private byte[] ecdsaPrivateToPublicKey(byte[] privateKeyBytes) {
+        final var ecdsaSecp256K1Curve = SECNamedCurves.getByName("secp256k1");
+        final var ecdsaSecp256K1Domain = new ECDomainParameters(
+                ecdsaSecp256K1Curve.getCurve(),
+                ecdsaSecp256K1Curve.getG(),
+                ecdsaSecp256K1Curve.getN(),
+                ecdsaSecp256K1Curve.getH());
+        final var privateKeyData = new BigInteger(1, privateKeyBytes);
+        var q = ecdsaSecp256K1Domain.getG().multiply(privateKeyData);
+        var publicParams = new ECPublicKeyParameters(q, ecdsaSecp256K1Domain);
+        return publicParams.getQ().getEncoded(true);
+    }
+
+    public List<HederaAccount> getBlockedAccountsCreated() {
```

Comment:
BTW, there is apparently no way currently to find out what these blocked accounts are via hashscan or a mirror node as neither synthetic account creation records nor memos are queryable.  There should really be a way to get this information out of a node, even if it is only in logs generated at bootup.  Above code, for example, just ignores things in the blocklist it can't properly convert or which already exist and there seems to be no way to find out externally what the actual state is.